tarodb/assets_store.go

```diff
@@ -1556,6 +1569,56 @@ func (a *AssetStore) LogPendingParcel(ctx context.Context,
 	})
 }
 
+// StoreProofDeliveryAttempt logs a proof delivery attempt to disk.
+func (a *AssetStore) StoreProofDeliveryAttempt(ctx context.Context,
+	locator proof.Locator) error {
+
+	var writeTxOpts AssetStoreTxOptions
+	return a.db.ExecTx(ctx, &writeTxOpts, func(q ActiveAssetsStore) error {
+		// Log proof delivery attempt and timestamp using the current
+		// time.
+		proofLocatorHash := locator.Hash()
+		err := q.InsertReceiverProofTransferAttempt(
+			ctx, InsertRecvProofTxAttemptParams{
+				ProofLocatorHash: proofLocatorHash[:],
+				TimeUnix:         time.Now().UTC(),
```

Comment:
FYI postgres will not return you the same value if you read this row because it has a granularity of Microseconds vs the Nonoseconds from this func

Reply:
Thanks for pointing that out yesterday.